python/lsst/scarlet/lite/component.py

```diff
@@ -151,6 +151,8 @@ def __init__(
         floor: float = 1e-20,
         monotonicity: Monotonicity | None = None,
         padding: int = 5,
+        is_symmetric: bool = False,
```

Comment:
Missing descriptions for these new params in the class docstring. Actually it looks like bg_thresh and floor are missing too.